sqlalchemy_imageattach/entity.py

```diff
@@ -179,14 +193,21 @@ def object_id(self):
         composite key.
 
         """
-        key_columns = inspect(type(self)).primary_key
-        pk = [c.name for c in key_columns if c.name not in ('width', 'height')]
+        pk = self.pk_names()
         if len(pk) == 1:
             pk_value = getattr(self, pk[0])
             if isinstance(pk_value, numbers.Integral):
                 return pk_value
         raise NotImplementedError('object_id property has to be implemented')
 
+    @property
+    def pk_data(self):
```

Comment:
It is an instance of `dict` but `_values` sounds like an instance of `list` for me. Is it ok?